src/xcode/ENA/ENA/Source/Services/PPAnalyticsSubmitter/PPAnalyticsSubmitter.swift

```diff
@@ -281,7 +286,7 @@ final class PPAnalyticsSubmitter: PPAnalyticsSubmitting {
 		   differenceBetweenTestResultAndCurrentDateInHours >= hoursSinceTestResultToSubmitKeySubmissionMetadata {
 			timeDifferenceFulfillsCriteria = true
 		}
-		return isSubmitted || timeDifferenceFulfillsCriteria
+		return (isSubmitted || timeDifferenceFulfillsCriteria) && isTestResultPositive
```

Comment:
why not return earlier  if `isTestResultPositive` is true?
I think all the stuff later isn't need for this criteria.

Reply:
Was think about this as well but this way we the code look exactly like the doc comment on top of the function :)